src/FAQ/Faq.js

```diff
@@ -1,14 +1,38 @@
 import React from 'react'
 import faqData from './FaqData'
+
+const Link = props =>
+    <a className="color-navigation font-weight-bold" target="blank" href={props.href}>
+        {props.children}
+    </a>
+
```

Comment:
unnötig, kann weg. 
`className` wird unten  in meinem Kommentar zur alten Zeile 11 hinzugefügt.